app/components/progress-bar.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { debounce } from 'lodash';
```

Comment:
- Any reason for adding such a `lodash` apart from using the debounce function it provides? 
Can we write this by ourself ?

Reply:
Removed loads and added a debounce method.
Just curious any other reason for this other than bundle size increase?

Reply:
- I personally prefer using a library if it solves your problem which you otherwise have taken you much longer to solve, apart from the bundle size adding a library also requires you to maintain it over the times as new there are new updated made to it. 

Having multiple libraries also increases your build time as there is more code for the bundler to go through.